.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: Java Test CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: Test with Gradle
+        shell: bash
+        run: cd almost-jira && ./gradlew test
```

Comment:
I would be good to also add uploading test report to GitHub artifactory using actions/upload-artifact and uploading file that will be generated in almost-jira/build/reports/tests/test/index.html

Reply:
maybe even all files in almost-jira/build/reports/ (this will make checking test results much easier)